extensions/security/deployment/src/test/java/io/quarkus/security/test/cdi/inheritance/SubclassDenyAllBean.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.security.test.cdi.inheritance;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class SubclassDenyAllBean extends DenyAllBean {
+}
```

Comment:
@manovotn I'd like to check I get it correctly. Here, we need `@Inherited` to have secured `ping` method declared on `DenyAllBean` that has a class-level `DenyAll` annotation? And if you added `pong` to `SubclassDenyAllBean` this still won't be secured based on `DenyAll` placed on the parent, correct?

that's what I expect, I'd just like to understand this: without `@Inherited` the `ping` method would not be secured, do I get it right?

Reply:
That's correct, yes.

Reply:
Cool, in that case changes indeed keep current behavior. Thanks.

Reply:
@michalvavrik thanks, let me check

Reply:
Ladislav beat me to it but yes, that's exactly the case.
I think I saw some other tests that verified this very behavior elsewhere so my tests focused just on the inheritance.